### ISSUE TITLE
Paths with whitespaces are broken

### ISSUE THREAD

**Issue by fabioticconi** [milaboratory/mixcr#71]
```
$ mixcr align -f --library local -r log_align.txt -l TRA -s mm "../../local/mol_med/tcr_cd3/150909 TC/150909-MID1_S1_L001_R1_001.fastq" mid1.vdjca
Exception in thread "main" java.io.FileNotFoundException: ../../local/mol_med/tcr_cd3/150909 (No such file or directory)
```

**Comment by mikessh** [comment]
Unfortunately, it looks like a problem with JCommander (https://github.com/cbeust/jcommander/issues/128). Is it so, @dbolotin ?

**Comment by dbolotin** [comment]
Yes, it is a problem with JCommander. I just updated it to 1.48 (milaboratory/mitools#7) and this seems to fix the problem. This fix will be included in 1.8 release.

@fabioticconi thanks for reporting!
